### ISSUE TITLE
Add option to toggle integer centering in LocationArgument and Location2DArgument

### ISSUE THREAD

**Issue by learliet** [JorelAli/CommandAPI#487]
### Description

Currently, the CommandAPI's `LocationType.PRECISE_POSITION` uses `Vec3Argument.vec3()`. This causes integer values to be centered, which may not always be desired. It seems that the `Vec3Argument` has a boolean option called "centerIntegers" that determines whether or not it performs the correction.

It would be beneficial to utilize this option, allowing for straightforward offset arguments that accept three numbers without any modifications. This enhancement would align with the behavior of the delta argument in the vanilla particle command.

### Extra details

A temporary workaround for developers who ran into the same issue:
```kotlin
val delta = args[0] as Location

val deltaNew = args.getRaw(0)!!.split(" ").map {
    if (it.contains('.')) null else it.toDoubleOrNull()
}.toMutableList()

delta.x = deltaNew[0] ?: delta.x
delta.y = deltaNew[1] ?: delta.y
delta.z = deltaNew[2] ?: delta.z
```

**Comment by JorelAli** [comment]
Awesome! I didn't know there was a parameter in the `Vec3Argument` class, and this finally solves the mystery that I had discovered in one of the CommandAPI's tests for the `LocationArgument`:

<img width="1023" alt="image" src="https://github.com/JorelAli/CommandAPI/assets/4613171/8fb30939-5b4e-4ebc-9452-a00a7c7ce6ec">

Will implement for the next release.

**Comment by JorelAli** [comment]
Implemented in 9.2.0.
